doc-checking-app-api/src/main/java/uk/gov/di/authentication/app/lambda/DocAppCallbackHandler.java

```diff
@@ -100,27 +103,38 @@ public APIGatewayProxyResponseEvent docAppCallbackRequestHandler(
                             try {
                                 var sessionCookiesIds =
                                         CookieHelper.parseSessionCookie(input.getHeaders())
-                                                .orElseThrow();
+                                                .orElse(null);
```

Comment:
Would it be possible to keep your orElseThrow() and then do the logging and exception as part of an inline lambda? Would save you separately checking nulls with `if` unless I've misunderstood (quite possible)

Reply:
@DTaylorGDS  I moved it away from the `orElseThrow` as I wanted to be able to differenciate between the cookie not being present and the session not being present and log different error messages for each scenario